hello.yml

```diff
@@ -0,0 +1,30 @@
+config:
+    target: 'http://localhost:3006'
+    http:
+        pool: 20
+    phases:
+      - duration: 30
+        arrivalRate: 20
```

Comment:
this is a pretty low rate, are you testing higher rates elsewhere?

Reply:
I'm using a loop. Each user performs 100 requests, for a total of 60,000 requests.